codespell_lib/data/dictionary.txt

```diff
@@ -386,25 +424,32 @@ adresable->addressable
 adresing->addressing
 adress->address
 adressable->addressable
+adresse->address
 adressed->addressed
 adresses->addresses
 adressing->addressing
+adrress->address
+adusted->adjusted
 advandced->advanced
 advane->advance
 advaned->advanced
+advanved->advanced
 adventrous->adventurous
 advertisment->advertisement
 advertisments->advertisements
 advesary->adversary
 adviced->advised
 adviseable->advisable
 advizable->advisable
+adwances->advances
 aeactivate->deactivate, activate,
 aequidistant->equidistant
 aequivalent->equivalent
 aeriel->aerial
 aeriels->aerials
 afair->affair
+afaraid->afraid
+afect->effect
```

Comment:
1st solution should be `affect`
https://en.oxforddictionaries.com/usage/affect-or-effect

Reply:
Ok. Anyway, what I do is to report the case I see, not what it could be.